components/merkledb/src/views/mod.rs

```diff
@@ -525,8 +429,12 @@ impl<T: IndexAccess> View<T> {
         }
     }
 
-    /// Inserts a key-value pair into the fork.
-    pub fn put<K, V>(&mut self, key: &K, value: V)
+    /// Crutch to be able to create metadata for indexes not present in the storage.
```

Comment:
I think that comments like this are useful for developers but not really appropriate for documentation.

Reply:
The `put_or_forget` method is crate-specific though, so the comment is *already* targeting MerkleDB devs. Would you like me to remove this comment? Make it a simple comment rather than a doc comment?

Reply:
Yeah, simple comment will be better IMO. However, that's completely not critical.